src/components/pagination/_pagination.scss

```diff
@@ -144,8 +145,7 @@ $css--helpers: true;
       height: 100%;
       border-left: 1px solid $ui-04;
       border-right: 1px solid $ui-04;
-      padding-left: 0.9rem;
-      padding-right: 0.9rem;
+      width: rem(42px);
```

Comment:
Just wanted to double-check if this change is intentional.

Reply:
Yes, thanks for checking! 😄  , removing the padding and adding the width makes the arrow centered inside the container without the need for spacing.